actors/account/src/lib.rs

```diff
@@ -82,7 +80,7 @@ impl ActorCode for Actor {
                 let addr = Self::pubkey_address(rt)?;
                 Ok(RawBytes::serialize(addr)?)
             }
-            None => Err(actor_error!(SysErrInvalidMethod; "Invalid method")),
+            None => Err(actor_error!(SysErrInvalidMethod; "Invalid method").into()),
```

Comment:
I get this is just prototyping, but I want only one of the `ActorError` or `Abort` types to survive. They're going after the same problem.

Reply:
They definitely should survive both in this construction. The conversion into an `Abort` is what triggers the abort, where as `ActorError` still can be used inside an actor to communicate recoverable errors.

Reply:
I don't think ActorError should be used for recoverable errors. I don't think there are use cases for that. I would say recoverable errors should not specify an abort code.

Reply:
where would that mapping to codes live then?